### DISCUSSION TITLE
How to make Sumatra the Defaultapp with -appdata ...

### DISCUSSION THREAD

**Discussion by Phaidros666** [sumatrapdfreader/sumatrapdf#4100] [Documentation]
Hello,
in my company sumatraPDF ist the Defaultapp. It is installed in C:/Programms/SumatraPdf ..., but the Settings are in %APPDATA/SumatraPDF.
Unfortunately I have no admin-rights.
If I start sumatra vs commandline: Sumatra.exe -appdata %APPDATA%/SumatraPDF, everything works fine, but how do I tell the Windows-Standart-App-Dialog to start Sumatra with options?
If I doubleclick a pdf, Sumatra starts without the -appdata option and I don't want to start Sumatra vs Sumatra.exe -appdata %APPDATA%/SumatrPdf and then open my pdf-file from Sumatra. Any Ideas? Greetings Georg

**Comment by GitHubRulesOK** [comment]
slightly unclear what you are attempting for a single user they have full rights to Microsoft recommended %appdatalocal% where all the support files will be located for any individual system users without admin rights.

If as Admin you install in recommended %programfiles% each user will follow the same recommended %appdatalocal% without admin rights.

The alternative is they each run a local portable app in a personal area where they have full personal rights.

Either way (installed or portable) you can redirect the -appdata folder to a personal area for personal use.

If your Admin has installed with files in %appdata%\sumatrapdf\ that would be a much older version and they should have ensured you have full rights to that folder !
